models/index.js

```diff
@@ -3,3 +3,4 @@ var mongoose = require("mongoose");
 // connect mongoose to your Mongo DB here
```

Comment:
I see you required mongoose in this file, but you don't actually use it here so you need to either setup the connection in this file, or move all of it over to server.js.